src/main/java/org/openrewrite/java/spring/StringOptimization.java

```diff
@@ -0,0 +1,27 @@
+package org.openrewrite.java.spring;
+
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import org.openrewrite.java.template.RecipeDescriptor;
+import org.springframework.util.StringUtils;
+
+@SuppressWarnings("ALL")
```

Comment:
This might be a bit broad

Reply:
Also be sure to add a `@RecipeDescriptor` to the outer class, as that'll get turned into a `Recipes` class itself, which needs documentation.